airflow/ui/vite.config.ts

```diff
@@ -19,38 +19,27 @@
 import react from "@vitejs/plugin-react-swc";
 import { defineConfig } from "vitest/config";
 
-// Replace the directory to work with the flask plugin generation
-const transformUrlSrc = () => {
-  return {
-    name: "transform-url-src",
-    order: "post",
-    transformIndexHtml(html: string) {
-      return html
-        .replace(`src="/assets/`, `src="/ui/assets/`)
-        .replace(`href="/`, `href="/ui/`);
-    },
-  };
-};
-
 // https://vitejs.dev/config/
 export default defineConfig({
-  plugins: [react(), transformUrlSrc()],
-  resolve: {
-    alias: {
-      openapi: "/openapi-gen",
-      src: "/src",
+  build: { chunkSizeWarningLimit: 1600, manifest: true },
+  plugins: [
+    react(),
+    // Replace the directory to work with the flask plugin generation
+    {
```

Comment:
Moved inline and made static because the output of the function was always the same.